src/components/modules/Notifications/Notifications.view.vue

```diff
@@ -1,3 +1,23 @@
+<template>
+  <div class="notify-container">
+    <notifications
+      :speed="500"
+      :duration="5000"
+      width="auto"
+      position="top center"
+      data-test="app-notification"
+      classes="notification app-notification"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'NotificationsView',
+};
+</script>
+
+<style lang="scss">
 // Flash notifications
 
 $notify-width: 600px;
```

Comment:
Create `root.scss` and define here `--notify-width`, then, reuse that instead scss variable